packages/material-ui/src/Tabs/Tabs.js

```diff
@@ -554,19 +556,11 @@ const Tabs = React.forwardRef(function Tabs(inProps, ref) {
     }
   });
 
-  React.useEffect(() => {
-    const handleResize = debounce(() => {
-      updateIndicatorState();
-      updateScrollButtonState();
-    });
-
-    const win = ownerWindow(tabsRef.current);
-    win.addEventListener('resize', handleResize);
-    return () => {
-      handleResize.clear();
-      win.removeEventListener('resize', handleResize);
-    };
-  }, [updateIndicatorState, updateScrollButtonState]);
+  const handleResize = debounce(() => {
```

Comment:
This creates a new instance at each render, it doesn't work, we also need to call `.clear()` on it to clean up the scheduled runs when unmounting.

Reply:
Thanks. It's addressed in this commit: https://github.com/mui-org/material-ui/pull/27791/commits/ba927d95f11996c0f7b92bf749c68e4f9df68cf3